client/src/pages/trainee/dashboard/index.tsx

```diff
@@ -0,0 +1,47 @@
+import Tabs from '@/src/shared/components/Tabs';
+import Tab from '@/src/shared/components/Tabs/Tab';
+import Link from 'next/link';
+import { Fragment } from 'react';
+import { type CustomSession } from '@/src/shared/utils/interface';
+import { useSession } from 'next-auth/react';
+
+const DashboardPage: React.FC = () => {
+  const { data } = useSession();
+  const session = data as CustomSession;
```

Comment:
```suggestion
  const { data:session } = useSession();
```

you can just destructure the assignment instead of creating a separate var